### PR TITLE
Mouse cursor images handling fix and cleanup

### PR DESCRIPTION
**Pull request by trunkmaster**
Fix for displaying of colored mouse cursor images.
Cleanup in Xlib cursor images loading - do not load cursors which handled by NSCursor.
Complementary to https://github.com/gnustep/libs-gui/pull/30.

### CODE REVIEW

File: Source/x11/XGServerWindow.m [hunk 1]
```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
     case GSIBeamCursor:
       cursor = XCreateFontCursor(dpy, XC_xterm);
       break;
-    case GSCrosshairCursor:
```
Comment (fredkiefer) [inline Source/x11/XGServerWindow.m]: Why did you remove this cursor?

File: Source/x11/XGServerWindow.m [hunk 2]
```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
...
-      cursor = XCreateFontCursor(dpy, XC_crosshair);
+    case GSClosedHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_fleur);
```
Comment (fredkiefer) [inline Source/x11/XGServerWindow.m]: Why use fleur for closed hand?

File: Source/x11/XGServerWindow.m [hunk 3]
```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
...
       break;
-    case GSDisappearingItemCursor:
```
Comment (fredkiefer) [inline Source/x11/XGServerWindow.m]: Why remove this?

File: Source/x11/XGServerWindow.m [hunk 4]
```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
...
-      cursor = XCreateFontCursor(dpy, XC_shuttle);
+    case GSOpenHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_hand1);
```
Comment (fredkiefer) [inline Source/x11/XGServerWindow.m]: For me this is rather a pointing hand.

File: Source/x11/XGServerWindow.m [hunk 5]
```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
...
     case GSPointingHandCursor:
-      cursor = XCreateFontCursor(dpy, XC_hand1);
+      cursor = XCreateFontCursor(dpy, XC_hand2);
+      break;
+    case GSCrosshairCursor:
+      cursor = XCreateFontCursor(dpy, XC_crosshair);
       break;
     case GSResizeDownCursor:
-      cursor = XCreateFontCursor(dpy, XC_bottom_side);
+      cursor = XCreateFontCursor(dpy, XC_sb_down_arrow);
```
Comment (fredkiefer) [inline Source/x11/XGServerWindow.m]: Sorry, but for all this resize images I prefer the old definitions. I might be old fashioned but this is what I expect for resize.

File: Source/x11/XGServerWindow.m (hunk 1 above)
Comment (trunkmaster) [reply Source/x11/XGServerWindow.m]: I didn't remove it. Look at line 4189.

File: Source/x11/XGServerWindow.m (hunk 2 above)
Comment (trunkmaster) [reply Source/x11/XGServerWindow.m]: Because XC_flue is the closest item to the closed hand. Anyway if you want to have closed hand cursor image you should use [NSCursor closedHandCursor]. It loads the right image (see appropriate PR in libs-gui). Probably these 2 lines hould be removed. What do you think?

File: Source/x11/XGServerWindow.m (hunk 4 above)
Comment (trunkmaster) [reply Source/x11/XGServerWindow.m]: Adwaita theme (standard de-facto) has an open hand. Look for the table I've created some time ago https://github.com/trunkmaster/nextspace/wiki/Mouse-Cursors

File: Source/x11/XGServerWindow.m (hunk 3 above)
Comment (trunkmaster) [reply Source/x11/XGServerWindow.m]: Because it loads by NSCursor as image.

File: Source/x11/XGServerWindow.m (hunk 5 above)
Comment (trunkmaster) [reply Source/x11/XGServerWindow.m]: I've reverted these changes.

**Comment by trunkmaster** [comment]
@fredkiefer, ping

File: Source/x11/XGServerWindow.m (hunk 2 above)
Comment (fredkiefer) [reply Source/x11/XGServerWindow.m]: I agree, removing these lines is less confusing.

Comment (trunkmaster) [reply Source/x11/XGServerWindow.m]: Done.
